toponetx/classes/combinatorial_complex.py

```diff
@@ -898,8 +965,8 @@ def singletons(self):
 
         Returns
         -------
-        singles : list
-            A list of cells uids.
+        singltons : list
```

Comment:
Singular returns don't have a time. The first line should only contain the type. (This is also validated by `numpydoc`)

```suggestion
       list
```